circle.yml

```diff
@@ -66,6 +67,7 @@ deployment:
             SENTRY_PROJECT: $DEMO_SENTRY_PROJECT
             SENTRY_EMBER_SOURCEMAPS_KEY: $DEMO_SENTRY_EMBER_SOURCEMAPS_KEY
             AUTH_SERVICE: $DEMO_AUTH_SERVICE
+            API_GATEWAY: $DEMO_API_GATEWAY
```

Comment:
please use `MEMBERSHIP_SERVICE` instead of `API_GATEWAY`, and then the value will be something like `'https://api.wnyc.org/membership'`